tests/phpunit/tests/auth.php

```diff
@@ -100,6 +100,31 @@ public function test_password_trimming() {
 		}
 	}
 
+	/**
+	 * Test hooking into wp_set_password().
+	 *
+	 * @ticket 57436
+	 */
+	public function test_wp_set_password_action() {
+		$user_id             = self::$user_id;
+		$expected_meta_value = 'Meta value';
```

Comment:
```suggestion
```

Remove variable. Check out my other comment regarding it.

Reply:
Thanks for the proposal, but it is now outdated since the test was entirely replaced with a MockAction() as per Sergey's suggestion :)